### PR TITLE
Manual.md: remove mentions of noarch, note it should not be used.

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

File: Manual.md [hunk 1]
```diff
@@ -725,9 +724,8 @@ Examples:
 	archs="x86_64-musl ~*-musl"
 	# Default value (all arches)
 	archs="*"
-	# Packages that do not depend on architecture-specific objects
-	archs=noarch
 	```
+Do not use noarch any longer. It is being deprecated and to be removed soon.
```
Comment (the-maldridge) [inline Manual.md:728]: Deprecation is an atomic action, removal takes time.   So this should read "It is deprecated and is being removed"

Comment (the-maldridge) [reply Manual.md:728]: Should probably nix 'any longer' as well.

**Review by the-maldridge — CHANGES_REQUESTED** [review]

File: Manual.md (hunk 1 above)
Comment (fosslinux) [reply Manual.md]: done

File: Manual.md [hunk 2]
```diff
@@ -914,8 +912,8 @@ can be used to pass arguments during compilation. If your package does not make
 extensions consider using the `gem` build style instead.
 
 - `gem` For packages that are installed using gems from [RubyGems](https://rubygems.org/).
-The gem command can be overridden by `gem_cmd`. `archs` is set to `noarch` unconditionally
-and `distfiles` is set by the build style if the template does not do so. If your gem
+The gem command can be overridden by `gem_cmd`. 
+`distfiles` is set by the build style if the template does not do so. If your gem
```
Comment (q66) [inline Manual.md:916]: this breaks the list

**Review by q66 — CHANGES_REQUESTED** [review]

File: Manual.md (hunk 2 above)
Comment (Chocimier) [reply Manual.md:916]: What do you mean? github renders it as 
```
<li>
<p><code>gem</code> For packages that are installed using gems from <a href="https://rubygems.org/" rel="nofollow">RubyGems</a>.
The gem command can be overridden by <code>gem_cmd</code>.
<code>distfiles</code> is set by the build style if the template does not do so. If your gem
provides extensions which must be compiled consider using the <code>gemspec</code> build style instead.</p>
</li>
```

Comment (q66) [reply Manual.md:916]: must have looked wrong, nevermind

Comment (fosslinux) [reply Manual.md:916]: afaict, there is no issue, is this ok?

**Comment by fosslinux** [comment]
ping?

**Pull request merged**